detection_rules/version_lock.py

```diff
@@ -171,7 +171,7 @@ def get_locked_hash(self, rule_id: str, min_stack_version: Optional[str] = None)
 
     def manage_versions(self, rules: RuleCollection,
                         exclude_version_update=False, save_changes=False,
-                        verbose=True) -> (List[str], List[str], List[str]):
+                        verbose=True, buffer_int: int = 99) -> (List[str], List[str], List[str]):
```

Comment:
@terrancedejesus Brough up the idea of incrementing by a percentage? like 9 -> 9.1. Thoughts?